src/main/java/frc/lib/util/swerveUtil/RevSwerveModuleConstants.java

```diff
@@ -0,0 +1,25 @@
+package frc.lib.util.swerveUtil;
```

Comment:
These constants are the same for Rev or Falcon.  Why was the class and file from https://github.com/TeamFusion364/PracticalFalconSwerve/blob/main/src/main/java/frc/lib/util/SwerveModuleConstants.java  renamed to have "Rev" at the start?  We can leave it with the original naming (and attribution)